test/isolate_test.dart

```diff
@@ -67,6 +67,14 @@ void main() {
       await main.setName('fblthp');
     });
 
+    test("reloadSources can succeed", () async {
+      var client = await runAndConnect();
+      var isolate = await (await client.getVM()).isolates.first.loadRunnable();
+
+      final report = await isolate.reloadSources(force: true);
+      expect(report.status, true);
+    }, tags: ["reload"]);
```

Comment:
We don't really use tags in this package, so tagging only this test is strange. In general, I don't use tags unless I have a specific reason to—either wanting to select a subset of tests that appear across many different files, or if I want to add special metadata for that tag.